Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs

```diff
@@ -258,6 +258,17 @@ protected override void Start()
             }
         }
 
+        protected override void OnDisable()
+        {
+            base.OnDisable();
+            this.gazeProvider = InputSystem.GazeProvider as GazeProvider;
+            BaseCursor c = gazeProvider.GazePointer.BaseCursor as BaseCursor;
+            if (c != null)
+            {
+                c.VisibleSourcesCount--;
```

Comment:
We might need some other bool state, or a comment saying that this is safe with this line:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/8f29ba829718dc366d9ad2371c21231fa68bf3ba/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs#L322

Otherwise it seems like it would be possible to drive the count down by two.

Reply:
Yeah, I only just noticed that other line. Currently looking into it.

Reply:
Yeah, line 319 should be:
```BaseCursor c = gazeProvider.GazePointer.BaseCursor as BaseCursor;```
That way we don't need the other decrement.